README.md

```diff
@@ -4,12 +4,16 @@
   <video src="https://user-images.githubusercontent.com/107984/157506173-5d2788e1-71ca-49ae-afc3-b7f7afbab798.mp4"></video>
 </div>
 
-*We asked an [AI](https://colab.research.google.com/github/zippy731/disco-diffusion-turbo/blob/main/Disco_Diffusion_v5_Turbo_%5Bw_3D_animation%5D.ipynb) what it thinks about our SocketCAN Kubernetes plugin, this was it's answer...  
+*We asked an [AI](https://colab.research.google.com/github/zippy731/disco-diffusion-turbo/blob/main/Disco_Diffusion_v5_Turbo_%5Bw_3D_animation%5D.ipynb) what it thinks about our SocketCAN Kubernetes plugin, this was it's answer...
```

Comment:
The double-space removal at the end is unintended, right?

For myself I disabled the automatic space removal at the end of line and instead installed a plugin that just highlights these spaces so I can decide myself. I guess disabling it for Markdown files would work as well.